test/functional/p2p_addr_relay.py

```diff
@@ -48,24 +53,72 @@ def run_test(self):
         self.log.info('Create connection that sends addr messages')
         addr_source = self.nodes[0].add_p2p_connection(P2PInterface())
         msg = msg_addr()
-
         self.log.info('Send too-large addr message')
-        msg.addrs = ADDRS * 101
+        msg.addrs = gen_addrs(1010, int(time.time()))
         with self.nodes[0].assert_debug_log(['addr message size = 1010']):
             addr_source.send_and_ping(msg)
 
         self.log.info('Check that addr message content is relayed and added to addrman')
         addr_receiver = self.nodes[0].add_p2p_connection(AddrReceiver())
-        msg.addrs = ADDRS
+        mocked_time = int(time.time())
+        self.nodes[0].setmocktime(mocked_time)
+        msg.addrs = gen_addrs(10, mocked_time)
         with self.nodes[0].assert_debug_log([
                 'Added 10 addresses from 127.0.0.1: 0 tried',
                 'received: addr (301 bytes) peer=0',
                 'sending addr (301 bytes) peer=1',
         ]):
             addr_source.send_and_ping(msg)
-            self.nodes[0].setmocktime(int(time.time()) + 30 * 60)
+            mocked_time += 30 * 60
+            self.nodes[0].setmocktime(mocked_time)
             addr_receiver.sync_with_ping()
 
+        self.log.info('Check that within 24 hours an addr relay destination is rotated at most once')
+        mocked_time = int(time.time()) + 100000
```

Comment:
I don't grok what/why regarding the 100000 value. It would be nice to comment and/or hoist it to a variable.

Same for 200000 at line 100.